pages/index.js

```diff
@@ -1,27 +1,109 @@
-import Card, {
-  CardHeader,
-  CardTitle,
-  CardContent,
-  CardButtonContainer,
-} from '../components/card';
-import { EDIT } from '../constants';
-import Button from '../components/button';
+import firebase from 'firebase'
+import styled from 'styled-components'
+import React, { useState } from 'react'
+import { useRouter } from 'next/router'
+import { COLOR } from '../constants'
+import { checkAdminClaim } from '../utility/auth'
+import background from '../public/backgroundImage.png'
+import nwPlusLogo from '../public/nwPlusLogo.png'
+import signIn from '../public/signIn.png'
+
+const BackgroundContainer = styled.div`
+    background-image: url(${background});
+    background-size: cover;
+    height: 100vh;
+    width: 100vw;
+`
+
+const LeftContainer = styled.div`
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    height: 100vh;
+    width: 337px;
+    background-color: white;
+`
+
+const LogoImage = styled.img`
+    align-self: center;
+    margin-top: 80px;
+`
+
+const LogInDiv = styled.div`
+    font-size: 28px;
+    margin-top: 48px;
+`
+
+const SignInImage = styled.img`
+    align-self: center;
+    margin-top: 34px;
+    cursor: pointer;
+`
+
+const StatusDiv = styled.div`
+    margin-top: 20px;
+    font-size: 26px;
+    text-align: center;
+    ${(props) => props.error && `color: ${COLOR.RED}`}
+`
 
 export default function Home() {
+  const router = useRouter()
+  const [ showError, setShowError ] = useState(false)
+  const [ isAddingClaim, setIsAddingClaim ] = useState(false)
+  const setAdmin = firebase.functions().httpsCallable('setAdmin')
+
+  const googleSignIn = async () => {
+    const provider = new firebase.auth.GoogleAuthProvider();
+    provider.setCustomParameters({
+      'hd': 'nwplus.io',
+      'prompt': 'consent'
+    })
+
+    try {
+      await firebase.auth().signInWithPopup(provider)
+      setShowError(false)
+      setIsAddingClaim(true)
+      const user = await firebase.auth().currentUser
+      const isAdmin = await checkAdminClaim(user)
+      if (isAdmin) {
+        await router.push('/landing')
+      } else {
+        const res = await setAdmin()
+        if (res.data.isAdmin) {
+          await user.getIdToken(true)
+          await router.push('/landing')
+        } else {
+          await firebase.auth().signOut()
+          await router.push('/')
+          setIsAddingClaim(false)
+          setShowError(true)
+        }
+      }
+    } catch (error) {
+      console.log(error.message)
+      alert(error.message)
+    }
+}
   return (
     <>
-      <div>Login Page</div>
-
-      <Card>
-        <CardHeader>
-          <CardTitle>Hi</CardTitle>
-          <p>Some extra text</p>
-          <CardButtonContainer>
-            <Button type={EDIT}>Hi there</Button>
-          </CardButtonContainer>
-        </CardHeader>
-        <CardContent>Example usage of card component</CardContent>
-      </Card>
+      <BackgroundContainer>
+          <LeftContainer>
```

Comment:
Why the left container? I think it looks fine, just wondering if you got it from design or something. If not, could you post a picture of it and tag design?

Reply:
nah I didnt get it from design. I used that as the flex container for everything on login page other than the background. Should I still ping design and what should I ask them?